apps/trip_plan/README.md

```diff
@@ -1,19 +1,17 @@
 # TripPlan
 
-**TODO: Add description**
+The TripPlan application provides trip plans for users of the site looking to explore potential routes from one place to another.
 
-## Installation
+## API
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `trip_plan` to your list of dependencies in `mix.exs`:
+This application relies on an API to a source of trip plan data. The API is defined as an abstract behavior so that we can can have different implementations. Currently we have `MockPlanner` which is used for tests, and `OpenTripPlanner` which calls out to an external resource that provides live data.
 
-```elixir
-def deps do
-  [{:trip_plan, "~> 0.1.0"}]
-end
-```
+### OpenTripPlanner
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/trip_plan](https://hexdocs.pm/trip_plan).
+We run our [own instance](https://github.com/mbta/OpenTripPlanner) of the [OpenTripPlanner (OTP)](https://www.opentripplanner.org) open source project. OTP returns [structured resources](http://dev.opentripplanner.org/apidoc/1.0.0/index.html) encoded as JSON.
 
+## Data Structures
+
+A [TripPlan](http://dev.opentripplanner.org/apidoc/1.0.0/json_TripPlan.html) consists of a list of [Itineraries](http://dev.opentripplanner.org/apidoc/1.0.0/json_Itinerary.html), or possible routes you might take.
+
+An Itinerary contains a sequenced list of [Legs](http://dev.opentripplanner.org/apidoc/1.0.0/json_Leg.html). Each leg is a portion of the journey on one mode. This could be "Take the Green Line to North Station," or, "Walk to the bus stop at the corner of Porter St. and Garfield Rd."
```

Comment:
Would it be useful to link to / document the internal representations of TripPlan/Itinerary/Leg? I don't recall if the Elixir API is a straight mapping to the OTP API, or if there's post-processing that happens.

Reply:
Good point, added those links as well.

Reply:
We could also include that each leg (if a transit one) contains the 3 different fare types (base, recommended and reduced) so that an estimation of the total price is also shown for every possible itinerary.

Reply:
@alicia-mbta Yep, great—added a paragraph for that as well.